TASK-3/Driver.cs

```diff
@@ -0,0 +1,26 @@
+namespace TASK;
+
+public class Driver : Person
+{
+    private DrivingLicense _drivingLicense;
+    private Vehicle _vehicle;
+
+    public DrivingLicense DrivingLicense
+    {
+        set => _drivingLicense = value;
+        get => _drivingLicense;
+    }
+
+    public Vehicle Vehicle
+    {
+        get => _vehicle;
+        set => _vehicle = value;
+    }
+
+    public Driver(DrivingLicense drivingLicense, Vehicle vehicle, Person person) : base(person.FirstName,
+        person.LastName, person.DateOfBirth)
+    {
+        DrivingLicense = drivingLicense;
+        Vehicle = vehicle;
+    }
```

Comment:
Constructor should comes before getters and setters after class fields

Reply:
Done.